src/layout-grid.scss

```diff
@@ -8,12 +8,28 @@
 
 $block: #{$fd-namespace}-layout-grid;
 
+.#{$block}-row {
+  margin-top: 1rem;
+
+  @media only screen and (max-width: 599px) {
+    margin-top: 0.5rem;
+  }
+}
+
 .#{$block} {
   @include fd-reset();
 
   display: grid;
-  grid-template-columns: repeat(3, 1fr);
-  grid-gap: var(--fd-width-gutter, $fd-width--gutter);
+  grid-template-columns: 100%;
+  grid-gap: 1rem;
+
+  @media only screen and (max-width: 599px) {
+    grid-gap: 0.5rem;
+  }
+
+  * {
+    height: fit-content;
```

Comment:
We shouldn't use `fit-content` in our styles. It is [not supported](https://caniuse.com/#search=fit-content) in many browsers